### ISSUE TITLE
Call for Maintainers

### ISSUE THREAD

**Issue by tylerstillwater** [stretchr/testify#152]
Hey everyone!

Life, as it tends to do, has progressed for me in such a way that I am no longer able to devote time to maintaining testify, especially as I no longer use it.

As such, this project has been severely neglected. For that, I apologize. I am hoping there is someone who would be willing to step up and take over maintaining this project so it gets the attention it deserves. If you would like to do so, please respond with a comment explaining a bit about yourself and what you would like to do as a maintainer of this project. I'll then add one or more of those users as collaborators.

Thanks everyone!

-Tyler

**Comment by DAddYE** [comment]
What are you using?

**Comment by tylerb** [comment]
I wanted something a bit simpler that also output errors in the same format as the standard testing framework, so I wrote [Is](github.com/tylerb/is).

**Comment by DAddYE** [comment]
I agree. IMO we should cleanup this (maybe a v2) and write something in between.

For example check this: https://github.com/stretchr/testify/issues/75#issuecomment-67374424

Autogenerate things will help a lot

**Comment by tylerb** [comment]
Agreed. Great stuff. I take it you're interested in becoming a maintainer?

**Comment by DAddYE** [comment]
Okay!

**Comment by tylerb** [comment]
Done. Go forth and maintain! ;)

**Comment by DAddYE** [comment]
Awesome! Thanks! I'll start on a new branch/tag!

**Comment by tylerb** [comment]
You're welcome. Thank you for being willing to step up and become a maintainer!

**Comment by DAddYE** [comment]
Thanks to you for this awesome package and the opportunity!

**Comment by ernesto-jimenez** [comment]
I'm really interested in the project, specially in the mocking library.

We are using it extensively in conjunction with `mockery` and `go generate`.

Happy to help if you need a hand. I was actually about to open an issue about some improvements to the mocking rather than just opening a pull request since I didn't know what the status was.

**Comment by tylerb** [comment]
Wonderful! Glad to hear it is working well for you! I'll add you when I get back to my computer. Thanks!

**Comment by Hanzei** [comment]
Hey guys,
I want to help getting testify back in shape. There are a lot of open issues, which needs to be dealt with. Also test coverage can be better and the overall toolchain is quite rusty.

Would you mind adding me as a maintainer?

**Comment by dineshkumar-cse** [comment]
Hi, we use testify extensively in all of our projects, Its so awesome. 
_Thanks for the mocks and asserts_ especially :) 
I've been following up the issues, and trying to raise PR for few. **I would love to be part of the maintainers community and to learn more and contribute**.

**Comment by ernesto-jimenez** [comment]
@Hanzei @dineshkumar-cse where are you guys based? It would be good to meet in Slack and have a chat :)

**Comment by dineshkumar-cse** [comment]
@ernesto-jimenez I'm from India. pinging you in gopers channel #dineshkumar :)

**Comment by timfeirg** [comment]
Hi guys, just checking the project status here, I made https://github.com/stretchr/testify/pull/594 a while ago but haven't managed to get any response from the project maintainers. Any advice on how I can push this forward?

BTW I noticed that @devdinu is actively engaging in recent PRs, are you still interested in becoming a member?

**Comment by devdinu** [comment]
@timfeirg Yes I'm, would love to see all the PR's getting merged, and bringing down the issues, and eventually writing v2 of testify.  I wouldn't prefer/like using someone else fork, the creators needs to have it, and we need more maintainers.

**Comment by timfeirg** [comment]
Good to know, perhaps you can contact some of the project members and become a maintainer yourself? @devdinu 

I wouldn't know how to appropriately contact them, I don't even know how to enter https://gophers.slack.com

**Comment by devdinu** [comment]
@timfeirg You can join by sending an [invite](https://invite.slack.golangbridge.org/).  You could also join #testify-dev channel so we can have informal conversation there.

I've already asked @ernesto-jimenez while back to add me and call for new maintainers, hopefully he will get it done when he's free.

**Comment by devdinu** [comment]
@ernesto-jimenez @matryer We still have issues and PR's coming up, Bumping up this again, would love to hear your thoughts and we need some actions to bring it back again.

**Comment by HaraldNordgren** [comment]
@ernesto-jimenez Would you be interested in having more maintainers? I use testify everyday in my work.

**Comment by KrzysztofMadejski** [comment]
I'd be happy to help as well.

**Comment by brackendawson** [comment]
I'm interested in helping, happy to triage bugs/PRs. I'm mainly interested in making sure v2 has a really good API by the time it gets officially tagged. Things like the `actual, expected` order being consistent (issue 399) and possibly writing some formal criteria to help decide if a proposed function should be in the API or not..

**Comment by dolmen** [comment]
I'm taking the torch for a while. Thanks to @ernesto-jimenez for promoting me.

**Comment by dolmen** [comment]
@brackendawson Are you still interested in co-maintaining? While triaging I've seen your valuable feedback on multiple PRs.

**Comment by HaraldNordgren** [comment]
Hi @dolmen!

I will happily help out to maintain the project. I have used the library for 7 years and have committed a number of approved fixes as well.

I have worked as a senior developer, tech lead and currently Head of Engineering.

Doing code reviews is and has been a part of my daily job for a long time 🤗

**Comment by brackendawson** [comment]
> @brackendawson Are you still interested in co-maintaining? While triaging I've seen your valuable feedback on multiple PRs.

Hi @dolmen, yes I'm still interested. Happy to keep triaging and start to cautiously merge things.

**Comment by brackendawson** [comment]
This call yielded two maintainers, and we have more moving up through the approver rank.
